host/3.0/publish.yml

```diff
@@ -1,334 +1,383 @@
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.6-buildenv
+
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.7
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.7-slim
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.7-appservice
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.7-appservice-quickstart
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.7-buildenv
+
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.8
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.8-slim
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.8-appservice
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.8-appservice-quickstart
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.8-buildenv
+
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.9
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.9-slim
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.9-appservice
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.9-appservice-quickstart
+        docker push $TARGET_REGISTRY/python:$(TargetVersion)-python3.9-buildenv
+
+        docker system prune -a -f
+      displayName: tag and push python images
+      continueOnError: false
+- stage: Nanoserver_Images
+  dependsOn: []
+  jobs:
+  - job: Publish
+    pool: 
+      vmImage: "windows-2019"
+    steps:
+    - bash: |
+        echo $pswd | docker login -u $(dockerUsername) --password-stdin azurefunctions.azurecr.io
+      displayName: login
+      continueOnError: false
+      env:
+        pswd: $(dockerPassword)
+
+    - bash: |
+        set -e
+        SOURCE_REGISTRY=azurefunctions.azurecr.io/azure-functions
+        TARGET_REGISTRY=azurefunctions.azurecr.io/public/azure-functions
+
+        if [ -z "$(TargetVersion)" ]; then
+          echo "ERROR: TargetVersion is required"
+          exit 1
+        fi
+
+        if [ -z "$(PrivateVersion)" ]; then
+          echo "ERROR: PrivateVersion is required"
+          exit 1
+        fi
+
+        echo "##vso[task.setvariable variable=SOURCE_REGISTRY]$SOURCE_REGISTRY"
+        echo "##vso[task.setvariable variable=TARGET_REGISTRY]$TARGET_REGISTRY"
+      displayName: set env
+      continueOnError: false
+    - bash: |
+        set -e
+        docker pull $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-nanoserver-1809
+
+        docker tag $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-nanoserver-1809 $TARGET_REGISTRY/dotnet:$(TargetVersion)-nanoserver-1809
+
+        docker tag $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-nanoserver-1809 $TARGET_REGISTRY/dotnet:3.0-nanoserver-1809
+
+        docker push $TARGET_REGISTRY/dotnet:$(TargetVersion)-nanoserver-1809
+
+        docker push $TARGET_REGISTRY/dotnet:3.0-nanoserver-1809
```

Comment:
lets move this step to part public images that happens after all the regions are rolled out

Reply:
Resolved thank you :)